test/test_parser.cpp

```diff
@@ -4441,6 +4454,19 @@ TEST(Parser_Dictionary)
     CHECK_EQUAL(message, "Sum not defined for timestamps");
 }
 
+TEST(Parser_DictionaryObjects)
+{
+    Group g;
+    auto dogs = g.add_table_with_primary_key("dog", type_String, "name");
+    auto persons = g.add_table_with_primary_key("person", type_String, "name");
+    auto col_dict = persons->add_column_dictionary(*dogs, "pets");
```

Comment:
There are CI warnings about these unused variables. I think I see where you are going with this test, but it is probably better in a separate fix.

Reply:
Yes, this is unfinished business. I got away from it. This will be done in another PR as you suggest.